enterprise-suite/values.yaml

```diff
@@ -78,6 +78,8 @@ imageCredentials:
 # Alertmanager ConfigMap. Set to the name of a ConfigMap, with a file `alertmanager.yml`.
 alertManagerConfigMap: alertmanager-default
 
+# Service type for exposing Console outside the cluster. Can be "NodePort", "LoadBalancer" or "false".
```

Comment:
Do we want to document `LoadBalancer` yet, cuz I'm assuming there's more to making that work than just setting `exposeServices: LoadBalancer`.  I could be wrong though...

Reply:
Yeah, it would also need an Ingress resource. Not sure we can set it up in our chart if exposeService==LoadBalancer, it needs to be configured for different network setups.